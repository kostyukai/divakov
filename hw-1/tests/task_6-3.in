135135.353153
