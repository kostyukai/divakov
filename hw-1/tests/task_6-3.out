0100000100000000011111101111101011010011010000011110000101001100
